templates/store/snap-details.html

```diff
@@ -255,9 +266,7 @@ <h4>Users by distribution (log)</h4>
       }
 
       if (typeof ClipboardJS !== 'undefined') {
-        var copy = new ClipboardJS('.js-clipboard-copy');
-
-        copy.on('success', function () {
+        snapcraft.copySuccess = function() {
```

Comment:
If we start using it as a global utility we should have it properly moved to own file/module. We shouldn't have channel map code dependent on a code that is inlined in the template.

Reply:
After you said about this, I looked in to it some more and it turns out clipboardjs uses event delegation, so no need to re-init after the initial page load